test/jdk/java/net/DatagramSocket/InterruptibleDatagramSocket.java

```diff
@@ -97,6 +98,9 @@ else if (!s.isClosed() && interruptible)
     }
 
     public static void main(String[] args) throws Exception {
+        if (Thread.currentThread().isVirtual()) {
+            throw new jtreg.SkippedException("skipping test execution through a virtual thread");
```

Comment:
Throwing .SkippedException is good but "through a virtual thread" looks a bit strange, maybe say that the main thread is a virtual thread or the test can only be run in a platform thread.

Reply:
Done - I've updated the PR to reword this message.